.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
     env:
     - TESTS=true
     - COVERAGE=true
+    - LINT=true
```

Comment:
what's the reason to separate this from the main `make lint` target? Does it run much slower?

Reply:
Interesting, I thought `make lint` wasn't being used on purpose, because it fails with:

```
$ make lint
go vet ./cmd/... ./model/... ./pkg/... ./plugin/... ./crossdock/... ./storage/... .
cmd/query/app/static_handler_test.go:121: possible formatting directive in Error call
exit status 1
make: *** [Makefile:114: lint] Error 1
```

This failure does not show up in Travis though. I just changed this PR to fix the lint failure above and to get `make lint` to also call the target `gas`